README.md

```diff
@@ -213,13 +213,79 @@ We use containers to minimize the installation requirements for development and
 
 We use Docker's Compose feature to spin up all application resources together. Please make sure you have Docker Compose installed on your system. You can find documentation on doing so here: [Docker Compose](https://docs.docker.com/compose/install)
 
-If the following commands return sensible results then Docker and Docker Compose are installed:
 
+#### Option 1: Using Podman (Recommended)
+
+1. **If transitioning from Docker Desktop**
+Remove any remaining Docker files/configurations. Further information on this below. 
```

Comment:
add - and this goes under "- Uninstall Docker Desktop (requires GUI uninstall and checking for remnants)"